projects/rero/ng-core/src/lib/menu/menu-widget/menu-widget.component.html

```diff
@@ -1,110 +1,99 @@
-              class="nav-link"
-              [ngClass]="item.getLabelAttribute('class')"
-            >
-              <ng-core-menu-widget-prefix-suffix [item]="item"></ng-core-menu-widget-prefix-suffix>
-            </span>
-          </ng-template>
-        </li>
-        <ng-template #withsubmenu>
-        <li
-          class="nav-item"
-          [ngClass]="{ 'dropdown': item.hasChildren() }"
-          dropdown
-        >
-          <a
-            class="nav-link"
-            [ngClass]="{ 'dropdown-toggle': item.hasChildren() }"
-            [attr.id]="item.getAttribute('id')"
-            [attr.data-toggle]="item.hasChildren() ? 'dropdown' : null"
-            dropdownToggle
-          >
-            <i *ngIf="item.hasExtra('iconClass')" [ngClass]="item.getExtra('iconClass')"></i>
-            <span *ngIf="!item.hasLabelAttribute('hideLabel')" class="ml-2">
-              <ng-core-menu-widget-prefix-suffix [item]="item"></ng-core-menu-widget-prefix-suffix>
-            </span>
-          </a>
-          <div *dropdownMenu
-            class="dropdown-menu"
-            [ngClass]="item.getAttribute('class')"
-            role="menu"
-          >
-            <ng-container *ngFor="let sub of item.getChildren()">
-              <a
-                *ngIf="sub.hasUri(); else subLinkRouterLink"
-                class="dropdown-item"
-                [attr.id]="sub.getAttribute('id')"
-                [attr.href]="sub.getUri()"
-                [attr.target]="sub.getAttribute('target')"
-                (click)="doClick(sub)"
-              >
-                <i *ngIf="sub.hasExtra('iconClass')" [ngClass]="sub.getExtra('iconClass')"></i>
-                <span *ngIf="!sub.hasLabelAttribute('hideLabel')" class="ml-2">
-                  <ng-core-menu-widget-prefix-suffix [item]="sub"></ng-core-menu-widget-prefix-suffix>
-                </span>
-              </a>
-              <ng-template #subLinkRouterLink>
-                <a
-                  class="dropdown-item"
-                  [attr.id]="sub.getAttribute('id')"
-                  [attr.target]="sub.getAttribute('target')"
-                  [routerLink]="sub.getRouterLink()"
-                  [queryParams]="sub.getQueryParams() ? sub.getQueryParams() : {}"
-                  (click)="doClick(sub)"
-                >
-                  <i *ngIf="sub.hasExtra('iconClass')" [ngClass]="sub.getExtra('iconClass')"></i>
-                  <span *ngIf="!sub.hasLabelAttribute('hideLabel')" class="ml-2">
-                    <ng-core-menu-widget-prefix-suffix [item]="sub"></ng-core-menu-widget-prefix-suffix>
-                  </span>
-                </a>
-              </ng-template>
-            </ng-container>
-          </div>
         </li>
+        <ng-template #withDropdown>
+          <li class="nav-item dropdown" dropdown placement="bottom right">
+            <a class="nav-link dropdown-toggle"
+               dropdownToggle
+               [routerLink]=""
```

Comment:
Why empty `routerLink` ?

Reply:
Without this, the "a" tag isn't recognize as a valid link by browser and the cursor stay as pointer (not the little hand).

Reply:
and with `href="#"` ?

Reply:
I think simply adding `href` alone works.